

Bilous et al. 2021https://www.biorxiv.org/content/10.1101/2021.06.07.447430v1.full.pdf

single-cell RNA sequencing (scRNA-seq) technologies offer unique opportunities 
for exploring heterogeneous cell populations. However, in-depth single-cell 
transcriptomic  characterization  of  complex  tissues  often  requires  profiling  tens  to 
hundreds of thousands of cells. Such large numbers of cells represent an important 
hurdle for downstream analyses, interpretation and visualization. Here we develop a 
network-based coarse-graining framework where highly similar cells are merged into 
super-cells. We demonstrate that super-cells not only preserve but often improve the 
results of downstream analyses including visualization, clustering, differential 
expression, cell type annotation, gene correlation, imputation, RNA velocity and data 
integration. By capitalizing on the redundancy inherent to scRNA-seq data, super-cells 
significantly facilitate and accelerate the construction and interpretation of single-cell 
atlases,  as  demonstrated  by  the  integration  of  1.46  million  cells  from  COVID-19 
patients in less than two hours on a standard desktop.

